content/6_Advanced/SCC.mdx

```diff
@@ -98,6 +97,71 @@ int main() {
 ```
 
 </CPPSection>
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class Main {
+	static final int N = 100010;
```

Comment:
why is this 10 greater than the actual N

Reply:
Fix it